backend/supabase/group_invites/functions.sql

```diff
@@ -0,0 +1,14 @@
+create
+or replace function get_last_week_long_link (this_group_id text) returns text as $$
+    SELECT id
+    FROM group_invites
+    WHERE (
+            group_id = this_group_id
+            -- just gets the last default link if it was generated less than 12 hours ago
+            and duration = '1 week'::interval
+            and max_uses is null
+            and created_time + duration > (now() + '6 days 12 hours'::interval)
```

Comment:
Seems like you could have just compared created_time without adding duration. Not a big deal though.